components/brave_wallet/browser/tx_meta.h

```diff
@@ -0,0 +1,67 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_WALLET_BROWSER_TX_META_H_
+#define BRAVE_COMPONENTS_BRAVE_WALLET_BROWSER_TX_META_H_
+
+#include <string>
+
+#include "base/time/time.h"
+#include "brave/components/brave_wallet/common/brave_wallet.mojom.h"
+
+namespace base {
+class Value;
+}  // namespace base
+
+namespace brave_wallet {
+
+class TxMeta {
+ public:
+  TxMeta();
+  virtual ~TxMeta() = default;
+  TxMeta(const TxMeta&) = delete;
```

Comment:
I removed it because of
```
../../brave/components/brave_wallet/browser/tx_meta.h:25:10: error: return type 'brave_wallet::TxMeta' is an abstract class
  TxMeta operator=(const TxMeta&) const;
```

Reply:
I forgot it is an abstract class, let's remove this in this abstract class then since it can't be instantiated.

Reply:
Deleted in https://github.com/brave/brave-core/commit/c57a9cac691671a4858c1b4c51df13356f16ae81